webui_runner/plugs/wamr_app_aot/echo_author.sh

```diff
@@ -0,0 +1,5 @@
+ORIG_CWD="$(pwd)"
+SCRIPT_LOC="$(dirname ${BASH_SOURCE:-$0})"
+cd ${SCRIPT_LOC}/build/lucet/
```

Comment:
`build/lucet` is probably a copy-paste typo; why don't we just have the author script in one place?

Reply:
And I guess the same idea applies to some of the other scripts as well: echo_commit.sh, echo_branch.sh, etc.

Reply:
👍 I'm guessing at the time we started with one and just did a copy exact. Also not sure if everything used git at the time .... there were about 6 or 7 VMs total at one point. But I agree, it is a good idea to share as much of this scripting as possible. The one challenge probably that the build folder will have a unique name but that can be figured out. Can we do this in a follow-up patch? It's logically separate from what this patch is trying to accomplish and affects more than just wamr.

Reply:
This can get fixed in https://github.com/bytecodealliance/sightglass/issues/32.